tracer/test/Datadog.Trace.Tests/Configuration/ImmutableAzureAppServiceSettingsTests.cs

```diff
@@ -196,45 +196,48 @@ public void NeedsDogStatsD(string value, bool expected)
         }
 
         [Fact]
-        public void GetIsAzureConsumptionPlanFunctionFalseWhenNoFunctionsEnvVars()
+        public void GetIsRunningMiniAgentInAzureFunctionsFalseWhenNoFunctionsEnvVars()
         {
             var settings = new ImmutableAzureAppServiceSettings(CreateConfigurationSource(), NullConfigurationTelemetry.Instance);
-            settings.IsFunctionsAppConsumptionPlan.Should().BeFalse();
+            settings.IsRunningMiniAgentInAzureFunctions.Should().BeFalse();
         }
 
         [Fact]
-        public void GetIsAzureConsumptionPlanFunctionFalseWhenNotConsumptionPlan()
+        public void GetIsRunningMiniAgentInAzureFunctionsFalseWhenNotConsumptionPlan()
         {
             var source = CreateConfigurationSource(
                 (ConfigurationKeys.AzureAppService.FunctionsWorkerRuntimeKey, "value"),
                 (ConfigurationKeys.AzureAppService.FunctionsExtensionVersionKey, "value"),
                 (ConfigurationKeys.AzureAppService.WebsiteSKU, "basic"));
 
             var settings = new ImmutableAzureAppServiceSettings(source, NullConfigurationTelemetry.Instance);
-            settings.IsFunctionsAppConsumptionPlan.Should().BeFalse();
+            if (Environment.OSVersion.Platform != PlatformID.Unix)
+            {
+                settings.IsRunningMiniAgentInAzureFunctions.Should().BeFalse();
+            }
```

Comment:
This should be true on nix right?
```suggestion
            var usesMiniAgent = Environment.OSVersion.Platform == PlatformID.Unix;
            settings.IsRunningMiniAgentInAzureFunctions.Should().Be(usesMiniAgent);
```